wiki/en/Running-a-Server.md

```diff
@@ -92,16 +92,14 @@ If you want to run a Server on a Raspberry Pi, you will need to [compile from so
 
 ### Installation
 
-1. Download the [latest headless .deb file]({{ site.download_root_link }}{{ site.download_file_names.deb-headless }})
+1. Download the latest headless .deb file: `wget {{ site.download_root_link }}{{ site.download_file_names.deb-headless }}`
```

Comment:
Is wget always installed? I thought some distros have curl instead (which is seen as more modern?)

Reply:
Yes. I googled it and it seems as if Ubuntu has wget by default while others have curl

Reply:
But Ubuntu has curl as well, so maybe we should specify using that. BTW the reason I originally had it as just "download" was to avoid getting sucked into tools and method arguments like this, but I agree it's hard to know where to draw the line.

Reply:
Can we assume a user knows about how to download the deb? I think that’s the main reason to specify or not to specify a command here.

Reply:
Well, we don't say on the Windows or iOS client pages "Use your web browser to download Jamulus...", so I assume that somebody interested in installing a headless server will know what they're doing to "acquire" the binary for the machine they wish to run it on.  After all, there's no mention of wget here: https://httpd.apache.org/download.cgi :-) 

My instinct here is to steer clear of anything that isn't strictly needed to be known in terms of using Jamulus though.

Reply:
Ok. So revert it:

```suggestion
1. Download the [latest headless .deb file]({{ site.download_root_link }}{{ site.download_file_names.deb-headless }})
```